code/API_definitions/geofencing.yaml

```diff
@@ -0,0 +1,658 @@
+openapi: 3.0.3
+info:
+  title: Device geofencing API
+  description: API to create, retrieve and delete event subscriptions to realize geofencing for a user device
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
```

Comment:
If we add security per operation, that will overwrite this one, so no need to add it.

Reply:
So delete these two lines: line 15-16 ?